### PR TITLE
CondDB frontierKey update

### PR DESCRIPTION
**Pull request by smorovic**
#### PR description:

frontierKey in OnlineDBOutputService and CondDBESSource is set to untracked string (this is a parameter relevant only at runtime).

Additional Python changes are included to add a VarParsing parameter and apply the comand line parameter to the OnlineDBOutputService.

#### PR validation:

Tested validity of Python changes with pylint.

### CODE REVIEW

**Comment by smorovic** [comment]
@cmsbuild please test

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
+1
11_3_X Backport Tested in Online DQM at P5 with new HLTD rpm

**Comment by yuanchao** [comment]
+1
* Modification on python configuration only (built ok)
* tests passed

**Comment by ggovi** [comment]
+1

File: DQM/Integration/python/clients/beam_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -385,6 +385,12 @@
             onlyAppendUpdatePolicy = cms.untracked.bool(True)
         ))
     )
+    try:
+        process.OnlineDBOutputService.frontierKey = cms.untracked.string(options.runUniqueKey)
+        print("Configured frontierKey", options.runUniqueKey)
+    except:
+        pass
```
Comment (silviodonato) [inline DQM/Integration/python/clients/beam_dqm_sourceclient-live_cfg.py:392]: @smorovic @cms-sw/dqm-l2 @cms-sw/alca-l2 @cms-sw/db-l2 isn't it better to add a message also in the "except" branch ?

Comment (smorovic) [reply DQM/Integration/python/clients/beam_dqm_sourceclient-live_cfg.py:392]: @silviodonato
with all possible input sources used in this and similar configuration files now having the runUniqueKey VarArgs option, we could also just remove try/except and set frontierKey within process.OnlineDBOutputService definition. Does this seem ok for @cms-sw/dqm-l2 and others?

Comment (jfernan2) [reply DQM/Integration/python/clients/beam_dqm_sourceclient-live_cfg.py:392]: Yes, fine from our side. Thanks

Comment (smorovic) [reply DQM/Integration/python/clients/beam_dqm_sourceclient-live_cfg.py]: Pushed the change. If it looks ok, I'll also add it to 11_3_X backport.

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
+1

**Comment by ggovi** [comment]
+1

**Comment by yuanchao** [comment]
+1
* frontierKey in OnlineDBOutputService and CondDBESSource set to untracked string; python module compiled ok
* Add a VarParsing parameter and apply CL parameter to OnlineDBOutputService (technical)
* WF matrix tests and unit tests passed

**Comment by silviodonato** [comment]
please test
to include the fix https://github.com/cms-sw/cmsdist/pull/6911

**Comment by qliphy** [comment]
+1

**Pull request merged**